tests/integration_tests/functional/test_api.py

```diff
@@ -795,11 +806,30 @@ def test_send_ctrl_alt_del(test_microvm_with_api):
 
 def _drive_patch(test_microvm):
     """Exercise drive patch test scenarios."""
-    # Patches without mandatory fields are not allowed.
-    expected_msg = "at least one property to patch: path_on_host, rate_limiter"
+    # Patches without mandatory fields for virtio block are not allowed.
+    expected_msg = "Invalid device type found"
```

Comment:
the more I think about it, the error message is not very descriptive. Is there a better way to guide users to the idea that they use inappropriate arguments to the call? Ideally, it should say something like "the arguments are not compatible with the device type".

Reply:
The full message is `Unable to patch the block device: Invalid device type found on the MMIO bus.` I can change it to be `Unable to patch the block device: Invalid device type found on the MMIO bus. Please verify the request arguments.` WDYT?

Reply:
works for me

Reply:
Added